src/Compilers/Core/Portable/CryptographicHashProvider.cs

```diff
@@ -200,5 +200,20 @@ internal static ImmutableArray<byte> ComputeHash(HashAlgorithmName algorithmName
                 return ImmutableArray.Create(incrementalHash.GetHashAndReset());
             }
         }
+
+        internal static ImmutableArray<byte> ComputeHash(SourceHashAlgorithm hashAlgorithm, ImmutableArray<byte> bytes)
```

Comment:
This two APIs seem like they would be good candidates to have `hashAlgorithm` be an optional parameter where the default value is our current preferred algorthimn. This would make the API easier to use as developers wouldn't have to know `SourceHashAlgorithmUtils.DefaultHashAlgorithm` was a thing (and I wouldn't have to mis-spell algorithm every single time)